src/webgl/shaders/line.vert

```diff
@@ -76,9 +78,11 @@ void main() {
     aTangentIn != aTangentOut
   ) ? 1. : 0.;
 
-  vec4 posp = uModelViewMatrix * aPosition;
-  vec4 posqIn = uModelViewMatrix * (aPosition + vec4(aTangentIn, 0));
-  vec4 posqOut = uModelViewMatrix * (aPosition + vec4(aTangentOut, 0));
+  vec4 position = abs(aSide) == 1. ? aFromPosition : aToPosition;
```

Comment:
Is there any way we can avoid all this branching? Maybe some clever math or predetermine the from / to position for the variable before reaching the shader?

Or is this just needless optimization?

Reply:
I'm not sure if I can think of a way to get from/to working before reaching the shader. I was hoping there would be a way to repeat attributes for a few vertices in an instance to use just one position attribute, but it looks like all we can do is repeat the attribute for every vertex in the instance.

I think I can make that happen with `mix()` to avoid the conditional! Not sure if that'll be faster or not but I can check and see.